components/style/servo/restyle_damage.rs

```diff
@@ -285,6 +276,13 @@ fn compute_damage(old: &ServoComputedValues, new: &ServoComputedValues) -> Servo
         get_inheritedbox.visibility
     ]);
 
+
+    // Paint worklets may depend on custom properties,
+    // so if they have changed we should repaint.
+    if old.get_custom_properties() != new.get_custom_properties() {
+        damage.insert(REPAINT);
```

Comment:
I think this could be too overkill... Could we do better? I guess this is fine for now.